django/db/backends/sqlite3/base.py

```diff
@@ -391,6 +391,10 @@ def _sqlite_datetime_parse(dt, tzname=None):
     except (TypeError, ValueError):
         return None
     if tzname is not None:
+        if '+' in tzname:
+            tzname = tzname.split('+')[0]
+        elif '-' in tzname:
+            tzname = tzname.split('-')[0]
```

Comment:
after that, I want to add a code block whose purpose is to convert the string to time. Is there another way to solve this?